export/build

```diff
@@ -89,13 +89,16 @@ def process_files(fs, chdir="../src"):
         def puttogether(hash, line):
             return "\\h{%s}  %s" % (hash, line) if hash else line
 
-        return "\n".join(puttogether(*a) for a in zip(checksums, highlighted.split("\n")))
+        return "\n".join(puttogether(*a) for a in zip(checksums, highlighted.split("\n"))).strip()
```

Comment:
Perhaps it would be more readable to write
```
highlighted = highlight(text).split('\n')
(...) prepend_hash(hash, line) for hash, line in zip(checksums, highlighted) (...)
```
```